src/item_location.cpp

```diff
@@ -738,3 +752,9 @@ void item_location::set_should_stack( bool should_stack ) const
 {
     ptr->should_stack = should_stack;
 }
+
+bool item_location::held_by( Character &who ) const
+{
+    return ( where() == type::character && g->critter_at<Character>( position() ) == &who ) ?
```

Comment:
i think i'd like this to be regular if statements for clarity instead of the nested ternary.

Reply:
Sure, will do.